src/components/ModelTableList/CloudGroup.js

```diff
@@ -67,48 +63,57 @@ export default function CloudGroup({ activeUser, filters }) {
       modelGroup.forEach((model) => {
         const { highestStatus } = getModelStatusGroupData(model);
         cloudModels.rows.push({
+          "data-test-model-id": model?.uuid,
           columns: [
             {
+              "data-test-column": "name",
               content: generateModelDetailsLink(
                 model.info.name,
                 model.info && model.info.ownerTag,
                 activeUser
               ),
             },
             {
+              "data-test-column": "owner",
               content: extractOwnerName(model.info.ownerTag),
             },
             {
+              "data-test-column": "status",
               content: generateStatusElement(highestStatus),
               className: "u-capitalise",
             },
             {
+              "data-test-column": "summary",
               content: getStatusValue(model, "summary"),
               className: "u-overflow--visible",
             },
             {
+              "data-test-column": "region",
               content: (
                 <a href="#_" className="p-link--soft">
                   {getStatusValue(model, "region")}
                 </a>
               ),
             },
             {
+              "data-test-column": "credential",
               content: (
                 <a href="#_" className="p-link--soft">
                   {getStatusValue(model.info, "cloudCredentialTag")}
                 </a>
               ),
             },
-            // We're not currently able to get the controller name from the API
-            // so, display the controller UUID instead.
             {
-              content: generateControllerUUID(
-                getStatusValue(model.info, "controllerUuid")
+              "data-test-column": "controller",
+              content: (
+                <span data-test="controller-name">
```

Comment:
Is it necessary to wrap this in a span and add another data attribute?

Reply:
Not at all. Leftover from before I know arbitrary attributes can do add.